packages/rspack/tests/Defaults.unittest.ts

```diff
@@ -259,6 +259,7 @@ describe("snapshots", () => {
 		  "output": {
 		    "assetModuleFilename": "[hash][ext][query]",
 		    "chunkFilename": "[name].js",
+		    "chunkLoadingGlobal": "webpackChunk@rspack/core",
```

Comment:
Look like it is not a valid identifier. It need use `to_identifier` to normalize it.

Reply:
ok. I add `to_identifier` in rust, so it looks no change in js.